client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -128,8 +153,10 @@ define( [ 'utils/utils' ], function( Utils ) {
                         '<th>Owner</th>' +
                         '<th># of Steps</th>' +
                         '<th>Published</th>' +
+                        '<th>Show in Tools panel</th>' +
                     '</tr></thead>';
             _.each( workflows, function( wf ) {
+                var checkbox_html = wf.show_in_tool_panel ? '<input type="checkbox" class="show-in-tool-panel" value="'+ wf.wf_id +'" checked="'+ wf.show_in_tool_panel +'">' : '<input type="checkbox" class="show-in-tool-panel" value="'+ wf.wf_id +'"';
```

Comment:
Can you remove the redundancy in this line? I t hink it is also missing a `>` the end for the second case.